apps/volunteer/main.py

```diff
@@ -16,17 +16,21 @@
     Shift,
     ShiftEntry,
 )
-from .init_data import venue_list, role_list, shift_list
+from .init_data import load_initial_venues, load_initial_roles, shift_list
 
 
 @volunteer.route("/")
 def main():
-    if (
-        feature_enabled("VOLUNTEERS_SCHEDULE")
-        and current_user.is_authenticated
-        and Volunteer.get_for_user(current_user)
-    ):
-        return redirect(url_for(".schedule"))
+    if current_user.is_authenticated and Volunteer.get_for_user(current_user):
+        volunteer = Volunteer.get_for_user(current_user)
+        if volunteer is None:
```

Comment:
I don't think `volunteer` can be `None` here (unless the volunteer is deleted in the DB between the two queries) or the check on line 24 would have failed. 

Could assign `volunteer` on line 24 with walrus: `if current_user.is_authenticated and (volunteer := Volunteer.get_for_user(current_user)):`